lib/vanagon/platform/rpm.rb

```diff
@@ -48,7 +48,7 @@ def rpm_defines
         defines =  %(--define '_topdir $(tempdir)/rpmbuild' )
         # RPM doesn't allow dashes in the os_name. This was added to
         # convert cisco-wrlinux to cisco_wrlinux
-        defines << %(--define 'dist .#{@os_name.gsub('-', '_')}#{@os_version}' )
+        defines << %(--define 'dist .#{@os_name.tr('-', '_')}#{@os_version}' )
```

Comment:
I had to double check, but `String#tr` was added in 1.9.x, so that's more than usable for our needs here.